arma-rs/src/lib.rs

```diff
@@ -49,21 +49,16 @@ pub mod testing;
 #[cfg(feature = "extension")]
 pub use testing::Result;
 
-#[cfg(all(windows, feature = "extension"))]
+#[cfg(all(feature = "extension"))]
 #[doc(hidden)]
 /// Used by generated code to call back into Arma
-pub type Callback = extern "stdcall" fn(
+pub type Callback = extern "system" fn(
     *const libc::c_char,
     *const libc::c_char,
     *const libc::c_char,
 ) -> libc::c_int;
-#[cfg(all(not(windows), feature = "extension"))]
-#[doc(hidden)]
-/// Used by generated code to call back into Arma
-pub type Callback =
-    extern "C" fn(*const libc::c_char, *const libc::c_char, *const libc::c_char) -> libc::c_int;
 /// Requests a call context from Arma
-pub type ContextRequest = unsafe extern "C" fn();
+pub type ContextRequest = unsafe extern "system" fn();
```

Comment:
I wasn't 100% sure why this was `"C"` even on windows
but I guess it's a func with no args/return so the calling convention doesn't matter?